internal/store/postgres/utils.go

```diff
@@ -46,3 +46,23 @@ func addOrderByClause(db *gorm.DB, conditions []string, options addOrderByClause
 		},
 	})
 }
+
+func addOrderBy(db *gorm.DB, orderBy string) *gorm.DB {
```

Comment:
any reason why we need this complex function just to handle order_by?

Reply:
in the API query we accept the order by parameter in the format like this: `?order_by=field_name:asc|desc` and order `asc|desc` is also optional, hence this function to parse that`